.gitignore

```diff
@@ -32,3 +32,5 @@ cache
 
 # connectors
 /packages/core/connectors
+
+fly.toml
```

Comment:
Better move under #misc or maybe give an extra comment indicating the purpose would be greate.